sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1310,12 +1347,12 @@ export class BlobClient extends StorageClient {
    * operation to copy from another storage account.
    * @see https://docs.microsoft.com/en-us/rest/api/storageservices/copy-blob
    *
-   * @param {string} copySource url to the ource Azure Blob/File.
+   * @param {string} copySource url to the source Azure Blob/File.
    * @param {BlobStartCopyFromURLOptions} [options] Optional options to the Blob Start Copy From URL operation.
    * @returns {Promise<BlobStartCopyFromURLResponse>}
    * @memberof BlobClient
    */
-  public async startCopyFromURL(
+  private async startCopyFromURL(
```

Comment:
>private async [](start = 2, length = 13)

Should move private methods to the tail of this class

constructor > public methods > private methods

Reply:
Will match storage styles!